src/udf/Cargo.toml

```diff
@@ -13,10 +13,15 @@ normal = ["workspace-hack"]
 [dependencies]
 arrow-array = { workspace = true }
 arrow-flight = { workspace = true }
+arrow-ipc = { workspace = true }
 arrow-schema = { workspace = true }
 arrow-select = { workspace = true }
+base64 = "0.21"
+bytes = "1.4"
 cfg-or-panic = "0.2"
 futures-util = "0.3.28"
+itertools = "0.11"
+risingwave_object_store = { workspace = true }
```

Comment:
I think it would be better to decouple UDF from object store and put related code into the compute node.